tests/suites/sanity/sanity_test.go

```diff
@@ -29,8 +31,9 @@ var (
 	TestInstance        = fmt.Sprintf("%s-instance", OperatorName)
 	KubeConfigPath      = os.Getenv("KUBECONFIG")
 	OperatorDirectory   = os.Getenv("OPERATOR_DIRECTORY")
-	// TODO(mpereira): read NodeCount from params.yaml.
-	NodeCount = 3
+
+	// Node Count of 1 for Sanity test to have the tests a little bit faster
+	NodeCount = 1
```

Comment:
With a node count of 1, the value provided by `assertNumberOfCassandraNodes` (which effectively gets the output from `nodetool status`) is greatly diminished, because now it's not testing that the nodes are forming a cluster at all.

Reply:
This is a very important and valuable test to maintain, IMO, as changes to KUDO or the KUDO C* operator itself might cause the cluster to not form anymore. This already happened before and the tests caught it.

Reply:
Yes, I do agree that it is important to check that the cluster is actually formed, but I think it should be enough to do this in the very last step of the sanity-test. Otherwise every Parameter-Update is triple the time as the rolling update to the cluster restarts every node.

Reply:
One could also argue that it's important to check the changes to the cluster like parameter updates happen to a formed cluster, not necessarily an isolated node. It feels to me that the benefit of having a faster integration test suite doesn't outweigh the downside of making the test checks less valuable.

The existing functions that only query 0th pods could also be improved, thinking about it...